microcloud/test/includes/microcloud.sh

```diff
@@ -910,6 +929,26 @@ setup_system() {
     else
       lxc exec "${name}" -- sh -c "PATH=\$PATH:/snap/bin snap install microcloud --channel latest/edge"
     fi
+
+
+    # Add debug binaries for MicroCloud.
```

Comment:
Can you please put these blocks for adding the debug binaries (MicroCloud and LXD) into its own functions so they can be called at both places?